src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,11 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        String result = "";
```

Comment:
Use StringBuilder instead of String. you could see this in [checklist](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html)